### PR TITLE
stuff

### PR DESCRIPTION
**Pull request by viktor76525**

### CODE REVIEW

**Review by sungeunbae — COMMENTED** [review]
Awesome. @viktor76525  Can you add some bits to README.md show-casing how you fixed the perspective issue with tilt options?  How many hours do you want to bill?

**Comment by viktor76525** [comment]
> Awesome. @viktor76525 Can you add some bits to README.md show-casing how you fixed the perspective issue with tilt options? How many hours do you want to bill?

Update README.md. A day as discussed is fine. Have you tested it? I ran into some issues with CPT files missing from the resources package.

**Comment by sungeunbae** [comment]
> > Awesome. @viktor76525 Can you add some bits to README.md show-casing how you fixed the perspective issue with tilt options? How many hours do you want to bill?
> 
> Update README.md. A day as discussed is fine. Have you tested it? I ran into some issues with CPT files missing from the resources package.

Gosh, I never received an alert for your reply. Sorry about delayed response. Will test it. Thanks

File: figures/nov2023.png
Comment (sungeunbae) [inline figures/nov2023.png:1]: Lovely!!! @viktor76525

**Pull request merged**